pkg/store/bucket_test.go

```diff
@@ -1113,9 +1113,9 @@ func createBlockWithOneSample(t testutil.TB, dir string, blockIndex int, totalSe
 	fmt.Println("Building block with numSeries:", totalSeries)
 
 	var series []storepb.Series
-	h, err := tsdb.NewHead(nil, nil, nil, 1, tsdb.DefaultStripeSize)
+	h, err := tsdb.NewHead(nil, nil, nil, 1, dir, nil, tsdb.DefaultStripeSize, nil)
 	testutil.Ok(t, err)
-	defer testutil.Ok(t, h.Close())
```

Comment:
I wonder how did this not cause an issue before :open_mouth:

Reply:
it's just a test (: